### ISSUE TITLE
Settings Not Working on Arc Browser

### ISSUE THREAD

**Issue by hardianlawi** [igrigorik/videospeed#1082]
Hi, the extension settings on Arc browser can't be saved for some reason. I tried on another browser like Brave and it worked just fine.

<img width="646" alt="image" src="https://github.com/igrigorik/videospeed/assets/22806872/a6b6dad4-4ef2-4c20-a0ab-16072a253a36">

This is what I see on my brave browser.

<img width="643" alt="image" src="https://github.com/igrigorik/videospeed/assets/22806872/0eed5447-1c50-4740-8045-9fc11f6f2de9">

This is what I see on Arc, but I can still use the shortcuts and the extension is working properly. Anyone can help to debug the issue?

**Comment by lucifermickelson** [comment]
Having the same issue, seems to be an arc specific issue so send  a feedback to them

**Comment by bphirsh** [comment]
Same issue. Has anyone found a fix?

I'd preference switching the plugin for another vs. abandoning Arc

**Comment by grandslammer** [comment]
> Same issue. Has anyone found a fix?
> 
> I'd preference switching the plugin for another vs. abandoning Arc.
> 
> Will modify comment if solved, if not plz reach out.

Same here. I stopped using Arc as soon as I discovered the issue. I looked for alternative extensions and there does seem to be a few available, but I haven't got around to trying any of them yet so I have no idea if any of them are as good as VSC.

If anyone has found a good alternative extension please let us know.

Also, can anybody confirm if this issue is affecting any other extensions or just VSC?

**Comment by grandslammer** [comment]
I found a similar extension that works with Arc.

https://chromewebstore.google.com/detail/nkkhljadiejecbgelalchmjncoilpnlk

**Comment by jorda0mega** [comment]
the issue is with Arc not having `chrome.storage` available. Either this extension would need to handle a special case for Arc to save the settings or Arc exposes the `chrome.storage` API to be used for this extension. I took a quick look and couldn't find an equivalent API for arc so I don't think a fix to this extension is trivial

**Comment by vitorhonna** [comment]
Same problem here. Any updates?

**Comment by github-throwaway** [comment]
@vitorhonna @grandslammer Is that extension still working for you? I just installed it and it didnt solve the issue for me :/

**Comment by vitorhonna** [comment]
> @vitorhonna @grandslammer Is that extension still working for you? I just installed it and it didnt solve the issue for me :/

Yes, it's still working for me. Did you remove the other extension?

**Comment by Historian9** [comment]
> > @vitorhonna @grandslammer Is that extension still working for you? I just installed it and it didnt solve the issue for me :/
> 
> Yes, it's still working for me. Did you remove the other extension?

how to open settings for VSC where you can configure shortcuts?

**Comment by jorda0mega** [comment]
> @vitorhonna @grandslammer Is that extension still working for you? I just installed it and it didnt solve the issue for me :/

it doesn't fix the settings issue for me either.

**Comment by ryankuah** [comment]
There are two work arounds, you can right click on the extension, and click options instead of left clicking on the extension, or reload the page after u got to settings. I believe the issues is that js is not loading after opening the page.

**Comment by torma616** [comment]
#1129 PR Submitted to fix this issue, should still work on all chrome/ium-based browsers
